From 6f7831516648cf70cb76d12a4b05c041279d40ba Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Mon, 21 Oct 2024 12:27:09 +0200
Subject: [PATCH] tools: Use esnacc instead of snacc in asn2deb

The snacc package is about to be removed from Debian, but esnacc,
a newer active fork from http://esnacc.org/ stays available.
---
 doc/man_pages/asn2deb.adoc | 4 ++--
 packaging/debian/control   | 2 +-
 tools/asn2deb              | 4 ++--
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/doc/man_pages/asn2deb.adoc b/doc/man_pages/asn2deb.adoc
index e02844ccf6..f043d27a89 100644
--- a/doc/man_pages/asn2deb.adoc
+++ b/doc/man_pages/asn2deb.adoc
@@ -33,7 +33,7 @@ asn2deb - Create a Debian package for BER monitoring from ASN.1
 This manual page documents briefly the *asn2deb* command. *asn2deb*
 takes an ASN.1 file as input and creates a Debian package from it. The package
 contains a loadable type table file for the Wireshark network analyzer.
-The type table is generated by the ASN.1 compiler *snacc*.
+The type table is generated by the ASN.1 compiler *esnacc*.
 
 == OPTIONS
 
@@ -65,7 +65,7 @@ print version and exit.
 == SEE ALSO
 
 A lot of tools are used, which you have to *apt-get install*: wireshark-dev, python, cdbs, autotools-dev, debhelper, dpkg-dev,
-snacc.
+esnacc.
 
 == COPYING
 
diff --git a/packaging/debian/control b/packaging/debian/control
index de03518dcf..d5a51c1eb1 100644
--- a/packaging/debian/control
+++ b/packaging/debian/control
@@ -125,7 +125,7 @@ Package: wireshark-dev
 Architecture: any
 Section: devel
 Depends: ${misc:Depends}, omniidl (>= 4.0.1-2), libpcap0.8-dev,
- libglib2.0-dev, python3:any, python3-ply, snacc,
+ libglib2.0-dev, python3:any, python3-ply, esnacc,
  libwireshark-dev, libwiretap-dev
 Description: network traffic analyzer - development tools
  Wireshark is a network "sniffer" - a tool that captures and analyzes
diff --git a/tools/asn2deb b/tools/asn2deb
index 926d34e2e5..7f0bbc78ef 100755
--- a/tools/asn2deb
+++ b/tools/asn2deb
@@ -4,7 +4,7 @@
 # Debian GNU/Linux packages from ASN.1 files for Wireshark.
 # Copyright 2004, W. Borgert
 
-# ASN.1 module for Wireshark, use of snacc type table:
+# ASN.1 module for Wireshark, use of esnacc type table:
 # Copyright 2003, Matthijs Melchior <matthijs.melchior@xs4all.nl>
 #
 # Wireshark - Network traffic analyzer
@@ -61,7 +61,7 @@ Section: net
 Priority: optional
 Maintainer: %s <%s>
 Standards-Version: 3.6.1.0
-Build-Depends: snacc, autotools-dev, debhelper, cdbs
+Build-Depends: esnacc, autotools-dev, debhelper, cdbs
 
 Package: wireshark-asn1-%s
 Architecture: all
-- 
2.43.0

